### DISCUSSION TITLE
Goal 7: Ensure access to affordable, reliable, sustainable and modern energy for all

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#29] [General]
Chat about anything related to Goal 7: [Ensure access to affordable, reliable, sustainable and modern energy for all](https://github.com/Project-Resilience/platform/blob/main/goals/goal_07.md)

**Comment by jnijmeh20** [comment]
A potential project idea: building an application to help decision-makers choose the right wind turbine, as the selection process is complex with many factors to consider. According to WindExchange (https://windexchange.energy.gov/small-wind-guidebook), criteria like turbine size, cost, maintenance, and wind conditions all need to be evaluated to make the best decision. Does anyone have any information to help?

**Reply by danyoungday** [reply]
Here's some data we can use from this wind turbine database! - [https://energy.usgs.gov/uswtdb/](https://energy.usgs.gov/uswtdb/)

**Reply by GPaolo** [reply]
another idea related to this, is the optimization of the wind farm usage. There is a new simulator/benchmark that came out at ICML last year that could help in that [https://arxiv.org/abs/2501.13592](https://arxiv.org/abs/2501.13592)
